### PR TITLE
Add toggle music feature

### PR DESCRIPTION
**Pull request by kedpak**
Added a toggle music feature. Music is toggled with the 'M' key. Added feature to controller instructions list.

### CODE REVIEW

**Comment by EdwardDunn** [comment]
@kedpak Thanks for opening a new PR. The feature isn't working currently, when 'm' is pressed the music does not mute, tested in Chrome and FireFox.

**Comment by kedpak** [comment]
Ah I think there was a typo in one of the functions. I will fix it.

File: Game.js [hunk 1]
```diff
@@ -65,8 +68,23 @@ function KeyDown(event) {
 		keysPressed[p]=false;
 		pauseGame();
 	}
+
+	if (keysPressed[m]) {
+	    keysPressed[m]=false;
+	    muteMusice();
```
Comment (kedpak) [inline Game.js]: Here this is a typo! I will fix

**Comment by EdwardDunn** [comment]
@kedpak Works great, great addition!

**Pull request merged**